### ISSUE TITLE
Motion zones no longer reflecting whether there is motion or not

### ISSUE THREAD

**Issue by amattas** [amattas/hass-dmp#17]
Motion zones are no longer returning the status indicating where there is motion or not.

**Comment by NebulaGazer** [comment]
Is this still an issue following the closure of #16

**Comment by amattas** [comment]
Yes, still an issue - haven't dove into debugging on it yet. I usually will err on opening the issue first otherwise I forget to go back to it. The joys of ADHD :-)

**Comment by amattas** [comment]
I changed this to an enhancement, I remember why I didn't implement this in the first place - I have battery powered motion sensors so they don't go into a sleep state when disarmed and it eats through batteries. I have implemented it regardless though as it's a useful feature.

**Comment by amattas** [comment]
@NebulaGazer or @krbaker can one of you take a look at the PR and code review it? It should be pretty straight forward.

**Comment by amattas** [comment]
Addressed by PR #27
